core-compose/src/androidTest/java/com/squareup/workflow/ui/compose/internal/ViewFactoriesTest.kt

```diff
@@ -48,14 +60,91 @@ class ViewFactoriesTest {
       WorkflowRendering(TestRendering("two"), viewEnvironment)
     }
 
-    findByText("one\ntwo").assertIsDisplayed()
+    onNodeWithText("one", useUnmergedTree = true).assertIsDisplayed()
+    onNodeWithText("two", useUnmergedTree = true).assertIsDisplayed()
+  }
+
+  @Test fun WorkflowRendering_rendersLegacyAndroidView() {
+    val viewEnvironment = ViewEnvironment(ViewRegistry(LegacyViewViewFactory))
+        .withCompositionRoot { content ->
+          Column {
+            Text("one")
+            content()
+          }
```

Comment:
This test doesn't need to assert anything about the composition root, it's unrelated and just makes this test more confusing.